Mithril update 2025-04-16 (#577)
IntersectMBO/cardano-updates: blog/2025-04-16-mithril.md

---
title: Mithril Team Update
slug: 2025-04-16-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team has started to roll out the incremental certification of the Cardano database (aka Cardano node database v2): it has been activated on the `pre-release-preview` test network.
> 
> They completed the adaptation of the infrastructure to support the aggregator’s prototype ‘follower’ mode. They also focused on signing ancillary files in the Cardano database snapshots with an IOG key. Additionally, they kept implementing the proof of concept for the Mithril/ALBA aggregation proof system.
> 
> Finally, the team continued refactoring the STM cryptographic library to improve clarity and worked on a refactoring of the commands configuration in the aggregator.

Reviewer: ```suggestion
This week, the Mithril team started rolling out incremental certification of the Cardano database (also known as Cardano node database v2), which has been activated on the `pre-release-preview test network`.

They also completed adapting the infrastructure to support the aggregator’s prototype ‘follower’ mode and focused on signing ancillary files in the Cardano database snapshots using an IOG key. Work also continued on the proof of concept for the Mithril/ALBA aggregation proof system.

Finally, the team continued refactoring the STM cryptographic library to improve clarity and started refactoring the aggregator’s command configuration.
```
